From d797136caed8782958c4796d902125faec9baf17 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 13 Jan 2021 17:38:20 +0800
Subject: [PATCH] ARM: rockchip_defconfig: Add CONFIG_UNICODE

Sync to gki_defconfig

Before:
    text	   data	    bss	     dec	    hex	filename
13465267	4345690	2113400	19924357	1300585	vmlinux

After:
    text	   data	    bss	     dec	    hex	filename
13473410	4411226	2113400	19998036	1312554	vmlinux

Change-Id: I023ce20549e9d65198b8a23affe4c9739f867103
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 289327583574..8bc4d4db5380 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -659,6 +659,7 @@ CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ASCII=y
 CONFIG_NLS_ISO8859_1=y
 CONFIG_NLS_UTF8=y
+CONFIG_UNICODE=y
 CONFIG_SECURITY_PERF_EVENTS_RESTRICT=y
 CONFIG_SECURITY=y
 CONFIG_SECURITY_NETWORK=y
-- 
2.35.3

